test/parallel/test-eventtarget.js

```diff
@@ -438,4 +439,36 @@ ok(EventTarget);
   strictEqual(target.toString(), '[object EventTarget]');
   const event = new Event('');
   strictEqual(event.toString(), '[object Event]');
```

Comment:
Looks like a missing `}` here?
```suggestion
  strictEqual(event.toString(), '[object Event]');
}
```